portal/templates/gil/base.html

```diff
@@ -37,15 +37,17 @@
     <meta name="msapplication-wide310x150logo" content="{{url_for('static', filename='img/widetile.png') }}">
     <meta name="msapplication-square310x310logo" content="{{url_for('static', filename='img/largetile.png') }}">
     <script>
-    var url = "{{url_for('user.login')}}";
-    $.ajax({
-        url: url,
-        cache: false,
-        success: function(response) {
-            result = $(response).find("#csrf_token");
-            $("#modal-login > div > div > div > form").append('<input id="csrf_token" name="csrf_token" type="hidden" value="'+ result.attr("value") +'">');
-        }
-    });
+      {% if not request.args.get("disableLinks", "") %}
+        var url = "{{url_for('user.login')}}";
+        $.ajax({
```

Comment:
Is it possible to use the CSRF token directly in the template here ([eg](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/templates/layout.html#L132))? This was a hack I originally added when we were hosting the GIL outside the portal codebase.

Reply:
Ivan, turns out we do currently send in the CSRF token in the form now.  (there was a round of fixes to add this back when we were patching up security risks of our system).  so I removed this ajax call - seems redundant now.